src/js/app.js

```diff
@@ -91,6 +91,49 @@ $(document).ready(function() {
     }
   });
 
+  //Add toggle effect for sliding #menu
+  let showingMenu = false;
+  const $menu = $('#menu');
+  const $main = $('main');
+  const config2 = {
+    duration: 1000,
+    easing: 'easeOut'
+  };
+  $('#menu-burger,.close-menu,#navigation-links>li>a').on('click', function() {
+    if (showingMenu) {
+      $menu.removeClass('active');
+      $main.removeClass('menu-active');
+      showingMenu = false;
+      event.preventDefault();
+    } else {
+      $menu.addClass('active');
+      $main.addClass('menu-active');
+      showingMenu = true;
+      event.preventDefault();
+    }
+    return showingMenu;
+  });
+
+  const burgerPos_y = $('#menu-burger').offset().top;
+  $(document).on('scroll', function() {
+    let scrollPos_y = $(window).scrollTop();
+    if (scrollPos_y + $(window).width() / 33 > burgerPos_y) {
+      // $('#menu-burger').css('transform', 'translateY(' + (scrollPos_y + ($(window).width() / 33.33) - burgerPos_y) + 'px)');
+      $('#menu-burger').css({
+        'position': 'fixed',
+        'top': '3vw',
+        'left': '3vw',
+        'margin': 0
+      });
+    } else {
+      // $('#menu-burger').css('transform', 'translateY(0)');
+      $('#menu-burger').css({
```

Comment:
In ideal world we keep HTML, CSS and JS seperate. Just like we don't use inline styles in HTML, you'd want to avoid adding CSS in JS. For animations `.animate()` and `.velocity()` it's unavoidable. However for `.css()` there is little good arguments.

Also as mentioned. Suppose you set `display: flex;` with `.css()` that will not be prefixed to `-webkit-flex` and `-ms-flex` variants. It's not JS' job  to prefix CSS.